bootstrap.sh

```diff
@@ -115,10 +115,12 @@ OS_REVISION=`echo $EVR	 | cut -d. -f3`
 
```

Comment:
Looks like this was missed earlier, but this file does not have an ALv2 header

Reply:
Thanks!